spec/models/form/sales/pages/shared_ownership_type_spec.rb

```diff
@@ -1,13 +1,30 @@
-require "rails_helper"
+require "rails_hel\per"
```

Comment:
What's happened here?

Reply:
interesting that it seems to have passed all tests anyway! maybe there is no need for rails_helper in certain test files???

Reply:
Yeah interesting, I'm actually more concerned with how I ended up committing that in the first place. But yes I guess this particular spec doesn't the rails_helper. So I guess I'll just remove it from this spec now that I know that